model/job-management/src/main/proto/org/apache/beam/model/job_management/v1/beam_job_api.proto

```diff
@@ -56,6 +56,9 @@ service JobService {
   // Cancel the job
   rpc Cancel (CancelJobRequest) returns (CancelJobResponse);
 
+  // Drain the job
+  rpc Drain (DrainJobRequest) returns (DrainJobResponse);
```

Comment:
Surprisingly enough drain feature wasn't a spec of Beam model. Although there us runner support it. This change worth note in CHANGE.md